test/components/FormDrawer.test.tsx

```diff
@@ -1,23 +1,5 @@
-import faker from 'faker';
-
-import { FormDrawer } from '../../src';
-import { Tester } from '@mighty-justice/tester';
-import SmartBool from '@mighty-justice/smart-bool';
-
 describe('FormDrawer', () => {
-  it('Renders', async () => {
-    const text = faker.lorem.sentence()
-      , title = faker.lorem.sentence()
-      , props = {
-        fieldSets: [[{ field: 'text' }]],
-        isVisible: new SmartBool(true),
-        model: { text },
-        title,
-      };
-
-    const tester = await new Tester(FormDrawer, { props }).mount();
-    expect(tester.text()).toContain(title);
-    expect(tester.text()).toContain('Text');
-    expect(tester.html()).toContain(text);
+  it('is tested the same as FormModal', async () => {
```

Comment:
Shouldn't this file be removed?

Reply:
I thought about it, but someone looking to test this component may not know where to look and I couldn't think of a better think to rename `FormModal.test.tsx` to indicate it tested both FormModal and FormDrawer.